content/en/blog/2024/cve-2024-36129.md

```diff
@@ -100,10 +100,11 @@ If you are using a custom distribution and building it with the
 can add a
 [“replaces”](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder#configuration)
 entry pointing to the latest version of the
-[confighttp](http://go.opentelemetry.io/collector/config/confighttp) and
-[configgrpc](http://go.opentelemetry.io/collector/config/configgrpc) Go module.
-If your base Collector version is at v0.96.0 or higher, we do not expect any
-compatibility issues by just bumping to the latest version.
+[confighttp](https://pkg.go.dev/go.opentelemetry.io/collector/config/confighttp)
+and
+[configgrpc](https://pkg.go.dev/go.opentelemetry.io/collector/config/configgrpc)
+Go module. If your base Collector version is at v0.96.0 or higher, we do not
+expect any compatibility issues by just bumping to the latest version.
```

Comment:
Note the change to use the canonical URLs here @open-telemetry/sig-security-maintainers.